_pages/vendor.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Vendor Team
+layout: page
+sidenav: false
+permalink: /vendors/
```

Comment:
@mgifford another singular/plural mismatch. Here it is `vendors` but the child menu link is `vendor`